action.yml

```diff
@@ -0,0 +1,57 @@
+name: Action for Building, Merging and Uploading Binaries to Github Pages - ESP Launchpad
+description: 'Builds and uploads binaries to github pages for ESP Launchpad'
+
+inputs:
+  github_token:
+    description: 'GitHub token'
+    required: true
+  idf_version:
+    description: 'IDF version'
+    required: false
+    default: 'latest'
+  parallel_count:
+    description: 'Number of parallel builds'
+    required: false
+    default: 1
+  parallel_index:
+    description: 'Index of the current parallel build'
+    required: false
+    default: 1
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Get Env variables
+      shell: bash
+      run: echo "IDF_VERSION=${{ inputs.idf_version }}" >> $GITHUB_ENV
+
+    - name: Fetch generateFiles.py
+      shell: bash 
+      run: |
+        curl \
+          -H "Accept: application/vnd.github.v3.raw" \
+          -o generateFiles.py \
+          -L https://api.github.com/repos/XDanielPaul/idf-examples-launchpad-ci-action/contents/generateFiles.py?ref=v1.0
```

Comment:
This path should be changed to new repository destination.

Reply:
This URL must not be static, it should be the same version as the action

Reply:
I removed the download of the script entirely and found out, that I can specify path to the script with `${GITHUB_ACTION_PATH}`